index.js

```diff
@@ -55,6 +57,7 @@ function ensureAllCommentsPrinted(astComments) {
 }
 
 function formatWithCursor(text, opts, addAlignmentSize) {
+  text = prependFormatIfNecessary(text);
```

Comment:
No worries, the cursorOffset issue is not terribly important until we want to merge this into upstream. Then it needs to be 100% correct. Would you mind just adding a TODO comment somewhere so that this issue is not forgotten?